backend/controllers/vendorControllers.js

```
@@ -0,0 +1,185 @@
+          });
+        }
+      });
+    }
+
+    // Append new image paths (if any) to existing ones 
+    if (newImagePaths) {
+      updatedImagePaths = [...updatedImagePaths, ...newImagePaths];
+    }
+    
+    // Parse amenities and availability from JSON string to object
+    let amenities, availability;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)